packages/sanity/src/core/comments/components/list/CommentsListItemLayout.tsx

```diff
@@ -193,6 +194,7 @@ export function CommentsListItemLayout(props: CommentsListItemLayoutProps) {
     onStatusChange,
     readOnly,
     withAvatar = true,
+    version,
```

Comment:
qq: we are receiving here the version but this is later not used, wdyt if we remove it and then add it back in case we want to update the designs?
I think it's more likely nothing will change design wise

Reply:
Yeah it's the latter (about waiting for design), mentioned in a todo on the same doc https://github.com/sanity-io/sanity/pull/7404/files#diff-68a543cbb2c056e1409c8f8453e10debce5d8f25bd1afda3da6933f26d886af5R341

I can just remove it if you prefer :)

Reply:
If you feel like it I'll appreciate it 😄 I think the less changes we add, specially to existing components, the easier the final review will be. 
Thanks!

Reply:
True, I'll go ahead and remove it :)